web3/gas_strategies/time_based.py

```diff
@@ -54,7 +60,11 @@ def _aggregate_miner_data(raw_data):
 
     for miner, miner_data in data_by_miner.items():
         _, block_hashes, gas_prices = map(set, zip(*miner_data))
-        yield MinerData(miner, len(set(block_hashes)), min(gas_prices))
+        yield MinerData(
+            miner,
+            len(set(block_hashes)),
+            min(gas_prices),
+            percentile(list(gas_prices), percentile=15))
```

Comment:
why the `list` cast?

Reply:
I will remove this.  I originally had pasted a percentile function from stackexchange that only accepted lists.  I replaced the function and didnt remove this.